.github/workflows/build-and-test.yml

```diff
@@ -33,12 +33,21 @@ jobs:
           - name: 'Linux x64'
             os: ubuntu-24.04
             task: 'linkReleaseExecutableLinuxX64'
+            artifact: 'osc-cli-linux-x64'
+            buildPath: 'cli/build/bin/linuxX64/releaseExecutable/osc.kexe'
+            binName: 'osc'
```

Comment:
Do you need the binary name as property, since it's always `osc`?

Reply:
For now, it is always `osc`. For Windows, it will be `osc.exe`, that's why I'm already making it configurable.